### ISSUE TITLE
[configure] Add support to use tao4 release

### ISSUE THREAD

**Issue by jwillemsen** [OpenDDS/OpenDDS#4636]

**Comment by jwillemsen** [comment]
The @RemedyIT test builds are now using ACE8/TAO4, we don't test anymore with ACE7/TAO3 from this point

**Comment by jwillemsen** [comment]
Looks https://github.com/OpenDDS/OpenDDS/actions/workflows/update-ace-tao.yml is disabled

**Comment by iguessthislldo** [comment]
> Looks https://github.com/OpenDDS/OpenDDS/actions/workflows/update-ace-tao.yml is disabled

Yes, I disabled it because it was constantly recreating its PR and I meant to re-enable it after #4594, but forgot. I did that just now (#4648), but it still won't do anything with ACE8/TAO4.

A new `ace8tao4` section is needed in `acetao.ini` and `gitrelease.pl` needs to be manually ran along with updates to `configure` and the [documentation](https://opendds.readthedocs.io/en/master/devguide/building/dependencies.html#ace-tao). CMake relies entirely on the `acetao.ini`, so I don't think it needs any changes except in [documentation](https://opendds.readthedocs.io/en/master/devguide/building/index.html#build-exclusive-cmake-variables). We should probably also make it so `configure` has an option like CMake's [`OPENDDS_ACE_TAO_KIND`](https://opendds.readthedocs.io/en/master/devguide/building/index.html#cmake-var-OPENDDS_ACE_TAO_KIND) and not an option like `--doc-group4`. That way in the future, at a minimum, only changes to `acetao.ini` and the documentation are needed.

**Comment by jrw972** [comment]
@iguessthislldo Can you expound on running `gitrelease.pl`?  Do I need to run all steps?  When should I do this in releation to the other tasks?

**Comment by iguessthislldo** [comment]
Sure, so what you'd do is add this to the `acetao.ini`:

```ini
[ace8tao4]
hold=0
desc=ACE 8/TAO 4
version=8.0.0
repo=https://github.com/DOCGroup/ACE_TAO.git
branch=master
```

Then the release script part is `gitrelease.pl --update-ace-tao`. It will fill in the additional details in `[ace8tao4]`.

> Do I need to run all steps? 

It doesn't run any release steps, it and a few other utilities just share a decent amount of code with the release script. It does need the normal release script Perl dependencies, but doesn't need `GITHUB_TOKEN` or other arguments like the version or workspace.

>  When should I do this in releation to the other tasks?

Everything else is going to be using `acetao.ini`, so I would suggest updating it first.
